packages/runtime/container-runtime/src/index.ts

```diff
@@ -20,6 +20,12 @@ export {
 } from "./containerRuntime.js";
 export type { ICompressionRuntimeOptions } from "./compressionDefinitions.js";
 export { CompressionAlgorithms, disabledCompressionConfig } from "./compressionDefinitions.js";
+export {
+	/**
+	 * @deprecated Import from `@fluidframework/runtime-utils/internal` instead.
```

Comment:
alpha+legacy API, right?
```suggestion
	 * @deprecated Import from `@fluidframework/runtime-utils/legacy` instead.
```